dotcom-rendering/src/layouts/NewsletterSignupLayout.tsx

```diff
@@ -445,20 +445,19 @@ export const NewsletterSignupLayout = ({ article, NAV, format }: Props) => {
 										/>
 									</div>
 
-									<SecureSignup
-										name={promotedNewsletter.name}
-										newsletterId={
-											promotedNewsletter.identityName
-										}
-										successDescription={
-											promotedNewsletter.successDescription
-										}
-										hidePrivacyMessage={true}
-									/>
-
-									<Hide from="desktop">
-										<NewsletterPrivacyMessage />
-									</Hide>
+									<Island
+										priority="feature"
+										defer={{ until: 'idle' }}
```

Comment:
```suggestion
										defer={{ until: 'visible' }}
```

Reply:
Un resolving as it's `visible` for the other Island. Why the distinction.

Reply:
Ah, there shouldn't be a distinction - they should both be visible. I've updated this